packages/auto-label/src/auto-label.ts

```diff
@@ -356,6 +356,70 @@ async function updateStalenessLabel(
   }
 }
 
+/**
+ * Function updating a T-shirt size label on pull request if configuration is enabled.
+ * Removes previous size label if exists
+ */
+async function updatePullRequestSizeLabel(
+  context: Context<'pull_request'>,
```

Comment:
nit: it's probably easier to pass in `octokit` rather than a context (you're not using any fields from the pull request).

Reply:
I also use `context.payload.pull_request.labels` below